egui_glow/Cargo.toml

```
@@ -23,11 +23,12 @@ include = [
 all-features = true
 
 [dependencies]
-egui = { version = "0.16.0", path = "../egui", default-features = false, features = ["single_threaded"] }
+egui = { version = "0.16.0", path = "../egui", default-features = false, features = ["single_threaded", "convert_bytemuck"] }
 
 epi = { version = "0.16.0", path = "../epi", optional = true }
 glow = "0.11"
 memoffset = "0.6"
+bytemuck = "1.7"
```

```suggestion
bytemuck = "1.7"
epi = { version = "0.16.0", path = "../epi", optional = true }
glow = "0.11"
memoffset = "0.6"
```

super-nit-picky I know, but I like to keep things nice and ordered